changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.86.1
+ * Fixed a bug which could cause crash when closed Realm in change listeners.
```

Comment:
Fixed a bug which could cause a crash when closing Realm instances in change listeners.